I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/level_4.rpy", line 332, in script
    if the_end.length > 1:
  File "game/level_4.rpy", line 332, in <module>
    if the_end.length > 1:
AttributeError: 'str' object has no attribute 'length'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/level_4.rpy", line 332, in script
    if the_end.length > 1:
  File "C:\Users\49157\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 2133, in execute
    if renpy.python.py_eval(condition):
  File "C:\Users\49157\Downloads\renpy-8.1.3-sdk\renpy\python.py", line 1153, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\49157\Downloads\renpy-8.1.3-sdk\renpy\python.py", line 1146, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/level_4.rpy", line 332, in <module>
    if the_end.length > 1:
AttributeError: 'str' object has no attribute 'length'

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.3.23091805
Social Engineering Game 1.0
Tue Jun 11 17:45:16 2024
